src/FS.Physical/baseline.netcore.json

```diff
@@ -88,51 +88,67 @@
           ],
           "Visibility": "Public",
           "GenericParameter": []
+        },
+        {
+          "Kind": "Constructor",
+          "Name": ".ctor",
+          "Parameters": [
+            {
+              "Name": "root",
+              "Type": "System.String"
+            },
+            {
+              "Name": "filters",
+              "Type": "Microsoft.Extensions.FileProviders.Physical.ExclusionFilters"
+            }
+          ],
+          "Visibility": "Public",
+          "GenericParameter": []
         }
       ],
       "GenericParameters": []
     },
     {
-      "Name": "Microsoft.Extensions.FileProviders.Internal.PhysicalDirectoryContents",
+      "Name": "Microsoft.Extensions.FileProviders.Physical.ExclusionFilters",
       "Visibility": "Public",
-      "Kind": "Class",
-      "ImplementedInterfaces": [
-        "Microsoft.Extensions.FileProviders.IDirectoryContents"
-      ],
+      "Kind": "Enumeration",
+      "Sealed": true,
+      "ImplementedInterfaces": [],
       "Members": [
         {
-          "Kind": "Method",
-          "Name": "get_Exists",
+          "Kind": "Field",
+          "Name": "Sensitive",
           "Parameters": [],
-          "ReturnType": "System.Boolean",
-          "Sealed": true,
-          "Virtual": true,
-          "ImplementedInterface": "Microsoft.Extensions.FileProviders.IDirectoryContents",
-          "Visibility": "Public",
-          "GenericParameter": []
+          "GenericParameter": [],
+          "Literal": "7"
         },
         {
-          "Kind": "Method",
-          "Name": "GetEnumerator",
+          "Kind": "Field",
+          "Name": "DotPrefixed",
           "Parameters": [],
-          "ReturnType": "System.Collections.Generic.IEnumerator<Microsoft.Extensions.FileProviders.IFileInfo>",
-          "Sealed": true,
-          "Virtual": true,
-          "ImplementedInterface": "System.Collections.Generic.IEnumerable<Microsoft.Extensions.FileProviders.IFileInfo>",
-          "Visibility": "Public",
-          "GenericParameter": []
+          "GenericParameter": [],
+          "Literal": "1"
         },
         {
-          "Kind": "Constructor",
-          "Name": ".ctor",
```

Comment:
The constructor which takes only a directory disappeared, is was this intentional?

Reply:
Nevermind, that's just the diff getting scrambled.